README_quality.md

```diff
@@ -0,0 +1,27 @@
+Quality code
```

Comment:
Instead of making a new file lets move this to be under https://github.com/internetarchive/openlibrary/blob/master/CONTRIBUTING.md#development-practices development practices.